src/CreateTwoDClustersFromThreeDAlgorithm.cc

```diff
@@ -23,6 +23,14 @@ CreateTwoDClustersFromThreeDAlgorithm::CreateTwoDClustersFromThreeDAlgorithm()
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+inline std::pair<int, int> QuanitizePosition(const float x, const float wirePos)
+{
+    constexpr float scale{1e-5};
```

Comment:
Instead of dividing by the scale, would it be faster to first define invScale{1e5} then use multiplication when making the pair integers? Also, please provide a short comment in the code about why this specific scale factor was chosen.

Reply:
Also, typo: Quantise -> Quantize?